<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Select_Path">
       <widget class="QPushButton" name="pb_start">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>460</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_path">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>50</y>
          <width>131</width>
          <height>29</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
       <widget class="QLabel" name="L_path">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>52</y>
          <width>541</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Please choose a path</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>801</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="L_username">
             <property name="text">
              <string>Enter User Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_username">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="L_">
             <property name="text">
              <string>       Enter Email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_email">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="L_path_4">
             <property name="text">
              <string>            Enter url</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_url">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="init">
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>411</width>
          <height>551</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_clear">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>550</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_back">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>550</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="L_currentpath">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>311</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Local</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>311</width>
           <height>253</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_status">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_log">
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="cb_add"/>
            </item>
            <item>
             <widget class="QPushButton" name="pd_add">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="cb_reset"/>
            </item>
            <item>
             <widget class="QPushButton" name="pb_reset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_diff">
            <property name="text">
             <string>Diff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_softreset">
            <property name="text">
             <string>Soft Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="2">
             <widget class="QPushButton" name="pb_commit">
              <property name="text">
               <string>Commit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="LE_commit">
              <property name="placeholderText">
               <string>Comment</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>290</y>
          <width>431</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Online</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="LE_renameold">
             <property name="placeholderText">
              <string>Old Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_renamenew">
             <property name="placeholderText">
              <string>New Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_rename">
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLineEdit" name="LE_removeremote">
             <property name="placeholderText">
              <string>Enter Name of remote you want to remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_remove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="LE_fetchalias">
             <property name="placeholderText">
              <string>Alias</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_fetch">
             <property name="text">
              <string>Fetch</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="LE_mergebranch">
             <property name="placeholderText">
              <string>Branch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_merge">
             <property name="text">
              <string>Merge</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="LE_remoteurl">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>URL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_remotealias">
             <property name="placeholderText">
              <string>Alias</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_remote">
             <property name="text">
              <string>Remote</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLineEdit" name="LE_pushalias">
             <property name="placeholderText">
              <string>Alias</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_pushbranch">
             <property name="placeholderText">
              <string>Branch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_push">
             <property name="text">
              <string>Push</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pb_pull">
           <property name="text">
            <string>Pull</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>111</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Branches</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>111</width>
           <height>102</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pb_delete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_checkout">
            <property name="text">
             <string>CheckOut</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_checkout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>110</width>
           <height>66</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="pb_createbranch">
            <property name="text">
             <string>Create Branch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_createbranch">
            <property name="placeholderText">
             <string>New Branch Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_back</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
